packit/ogr_services.py

```diff
@@ -0,0 +1,11 @@
+import inspect
+from packit.exceptions import PackitException
+from ogr.services.github import GithubService as GithubServiceOrigin
+from ogr.services.pagure import PagureService as PagureServiceOrigin
+
+
+if "read_only" not in inspect.signature(GithubServiceOrigin).parameters:
```

Comment:
But this is literally what I'm trying to prevent: don't fail if dry_run=False and read_only is not in GithubServiceOrigin. This is what I had in mind:
```python
if dry_run:
  if "read_only" not in inspect...:
    raise
  else:
    # set up dry run
else:
  # business as usual, no errors
```

test results now:
```
$ tox
py36 runtests: commands[0] | pytest --color=yes --verbose --showlocals --cov=packit --cov-report=term-missing
ImportError while loading conftest '/home/tt/g/user-cont/packit/tests/conftest.py'.
tests/conftest.py:38: in <module>
    from packit.api import PackitAPI
packit/api.py:35: in <module>
    from packit.distgit import DistGit
packit/distgit.py:29: in <module>
    from packit.ogr_services import PagureService
packit/ogr_services.py:8: in <module>
    raise PackitException("Read only mode is not supported by ogr library")
E   packit.exceptions.PackitException: Read only mode is not supported by ogr library
ERROR: InvocationError for command '/home/tt/g/user-cont/packit/.tox/py36/bin/pytest --color=yes --verbose --showlocals --cov=packit --cov-report=term-missing' (exited with code 4)
```

Reply:
@TomasTomecek  yes, I've found that after I've submitted commit.
the next commit add more magic and do it more dynamical.
and just avoid that in case it is really needed.

I've also tough to check support in cli option when dryrun is enable, but maybe this is nonsense in case we don't need so that I've added there decorator what wrapped call of classes